bucket/sikulix.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "2.0.5",
+    "description": "Visual Automation and Testing",
+    "homepage": "https://sikulix.github.io/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://launchpad.net/sikuli/sikulix/2.0.5/+download/sikulixide-2.0.5.jar",
+            "hash": "f4b0b50c8e413094e78cd1d8fed02ae65f62f8c53ed00da0562fdedf4acff729",
+            "bin": [
+                ["java.exe", "sikulix", "-jar $dir\\sikulixide-2.0.5.jar"]
+            ]
+        }
+    },
+    "checkver": {
+        "url": "https://raiman.github.io/SikuliX1/downloads.html",
+        "regex": "Version \\d[.]\\d[.]\\d (latest stable version)"
+    },
+    "autoupdate": {
+        "url": "https://launchpad.net/sikuli/sikulix/$version/+download/sikulixide-$version.jar"
+    }
```

```suggestion
    "checkver": {
        "url": "https://raiman.github.io/SikuliX1/downloads.html",
        "regex": "Version ([\\d.]+) \\(latest stable version\\)"
    },
    "autoupdate": {
        "url": "https://launchpad.net/sikuli/sikulix/$version/+download/sikulixide-$version.jar",
        "bin": [
            [
                "java.exe",
                "sikulix",
                "-jar",
                "\"$dir\\sikulixide-$version.jar\""
            ]
        ]
    }
```
BTW you can test autoupdate by typing
```pwsh
& "$Env:SCOOP\apps\scoop\current\bin\checkver.ps1" -Dir . -ForceUpdate sikulix
```

---

@ftk  Awesome! Thank you so much! I will test your suggested change with your suggested method as soon as I can. Thanks again.